source/retryable-writes/retryable-writes.rst

```diff
@@ -262,6 +262,10 @@ server may not support retryable writes if the
 cluster; however, that can only be reported as a server-side error (discussed
 later).
 
+Write commands that would be executed on a standalone server do not support
+retryable behavior. Drivers MUST NOT add a transaction ID to any write command
+that would be sent to a standalone and MUST NOT retry these commands.
```

Comment:
In [SPEC-969](https://jira.mongodb.org/browse/SPEC-969?focusedCommentId=1703044&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1703044), @ShaneHarvey asked if we'd like to raise an error (similar to bad wire protocol version) if the server is a standalone or just act as if `retryWrites=false`. This text assumes the latter.

Reply:
@kaloianm suggested that we rely on the server to raise an error in this case, so I've noted that in the SPEC ticket and amended this text.